themes/gatsby-theme-blog/src/theme/prism.js

```diff
@@ -0,0 +1,48 @@
+// todo: swap for theme colors
+export default {
+  ".attr-name": {
+    fontStyle: `italic`,
+  },
+  ".comment": {
+    color: `prism.comment`,
+  },
+  [[`.attr-name`, `.string`, `.url`]]: {
```

Comment:
Do you know why this requires a nested array?

Reply:
This is sorta weird syntax and isn't entirely necessary; the outer brackets are for the object key and the inner brackets are an array. Emotion will convert this to something like `.hashed-classname .attr-name, .hashed-classname .string` – it might be more readable to use a single string key here instead, which I think Emotion handles the same way